Programs/_testembed.c

```diff
@@ -1733,7 +1733,7 @@ static int check_use_frozen_modules(const char *rawval)
     if (rawval == NULL) {
         wcscpy(optval, L"frozen_modules");
     }
-    else if (swprintf(optval, 100, L"frozen_modules=%s", rawval) < 0) {
+    else if (swprintf(optval, 100, L"frozen_modules=%S", rawval) < 0) {
```

Comment:
```
       S      (Not in C99 or C11, but in SUSv2, SUSv3, and SUSv4.)  Synonym for ls.  Don't use.
```

Reply:
:( Looks like we need to revert this then.

Reply:
Or at least, guard it for Windows-only.

Reply:
I feel dumb now 🤦 . It seems there is a legit fix using C99 standard in https://github.com/python/cpython/pull/29341

Reply:
Thanks, both of you! 🤝